<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserTable</class>
 <widget class="QDialog" name="UserTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>650</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>92</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>30</number>
   </attribute>
   <column>
    <property name="text">
     <string>Player_name</string>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Birthday_date</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Game</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Goal</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Assist</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Point</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Penalty_Minutes</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="top_fife_player_button">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TOP 5 Player</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>691</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(16, 30, 41, 240);
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logout_button">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>390</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Log Out</string>
   </property>
  </widget>
  <widget class="QPushButton" name="player_list_button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>390</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Player List</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>140</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Player Name</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Goals</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Games</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="sorting_button">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>20</y>
     <width>150</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Sorting</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="find_player_field">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>181</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0 0);
font: 14pt &quot;MS UI Gothic&quot;;
border:1px solid rgba(0, 0, 0, 0);
border-bottom-color:rgba(46, 82, 101, 255);
color:rgb(255, 255, 255);
padding-bottom:7px</string>
   </property>
   <property name="placeholderText">
    <string>Find Player</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>tableWidget</zorder>
  <zorder>top_fife_player_button</zorder>
  <zorder>logout_button</zorder>
  <zorder>player_list_button</zorder>
  <zorder>comboBox</zorder>
  <zorder>sorting_button</zorder>
  <zorder>find_player_field</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
